encrypt-mail-ru.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+
+echo "Mounting Mail.ru Cloud enqrypted..."
+marcfs /tmp/mailru -o username=$MAILRU_LOGIN,password=$MAILRU_PASSWORD
+mkdir -p  /tmp/mailru/encrypted
+echo GdnAvRbs4yf7dNbs5eXD | encfs -S /tmp/mailru/encrypted /tmp/decrypted
```

Comment:
Do you mean, we have your private key here? ;)

Reply:
Fortunately not (don't use this image alredy), but the one i tested with. 
I hope this stays between us.